static/css/footer.less

```diff
@@ -44,101 +48,33 @@
             .u-link__colors(@black);
 
             .respond-to-min(768px, {
```

Comment:
Good catch on that. Although I just noticed that footer_nav-list doesn't have any border and really doesn't need this media query at all. Fixing!